### DISCUSSION TITLE
Parquet support

### DISCUSSION THREAD

**Discussion by coou** [kuzudb/kuzu#1263] [Q&A] [answered]
Is it on your roadmap to include support for Apache Parquet file serialization format?

**Comment by ray6080** [comment]
Hi @coou , thanks for your interest! The support of parquet file format is in our roadmap.

Currently in our latest master, we've supported data import from parquet files, specifically, you can COPY from a parquet file into a node or rel table. We will release a version with this feature soon.
Later, we will support export to parquet files, too.

Please let us know if you have any suggestions or potential use cases around this.

**Comment by prrao87** [comment, marked as answer]
@coou Parquet support for both read/write are available in Kùzu. See the "Import data" and "Export data" section of the docs:
https://docs.kuzudb.com/
